src/index.html

```diff
@@ -1,21 +1,78 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta 
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+
+      <div class="container__card card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <div class="card__product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+
+        <div class="card__reviews">
+
+          <div class="stars stars--4">
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+          </div>
```

Comment:
```suggestion
        <div class="card__reviews">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```

add empty line only between multiline siblings elements